src/nativestore/CentralNodeBlock.cpp

```diff
@@ -0,0 +1,281 @@
+//
```

Comment:
We need to use proper JasmineGraph project's copyright header here. Please refer other files and update accordingly.

Reply:
Done.

Reply:
Ok